shortnames.conf

```diff
@@ -163,3 +163,5 @@
   "qubip/pq-container" = "quay.io/qubip/pq-container"
   # SonarQube
   "sonarqube" = "docker.io/library/sonarqube"
+  # Aqua Security
+  "aquasec/trivy" = "docker.io/aquasec/trivy"
```

Comment:
Looking at how podman and the other tools reading shortnames behaves, I think t would be better to use "trivy" as the key.

The tools can already resolve "aquasec/trivy" to "docker.io/..."  but "trivy" would resolve to "docker.io/library/trivy" which does not exist.